data/lib/core/achievements.lua

```diff
@@ -0,0 +1,659 @@
+--[[
+Achievements Lib Created By Eduardo Montilva (Darkhaos)
+
+Functions:
+	getAchievementInfoById(achievement_id)
+	getAchievementInfoByName(achievement_name)
+	getSecretAchievements()
+	getPublicAchievements()
+	getAchievements()
+	Player:addAchievement(achievement_id/name[, showMsg])
+	Player:removeAchievement(achievement_id/name)
+	Player:hasAchievement(achievement_id/name)
+	Player:addAllAchievements([showMsg])
+	Player:removeAllAchievements()
+	Player:getSecretAchievements()
+	Player:getPublicAchievements()
+	Player:getAchievements()
+	isAchievementSecret(achievement_id/name)
+	Player:getAchievementPoints()
+	Player:addAchievementProgress()
+]]
+
+-- base storage
+ACHIEVEMENTS_BASE = 300000 
```

Comment:
data/lib/core/storages.lua ?

Reply:
since it will increase by itself, how will it work?

Reply:
Just the base should be there with proper comment that a lot of them from that starting point to X are used.
What's the point of central place to store storage IDs otherwise?